.github/workflows/environment.yml

```diff
@@ -17,11 +17,16 @@ jobs:
     env:
       ENVIRONMENT_AUTOBUILD: true
     steps:
-      - run: docker login https://docker.pkg.github.com -u ${{ github.actor }} -p ${{ secrets.GITHUB_TOKEN }}
+      - run: |
+          docker login https://docker.pkg.github.com -u ${{ github.actor }} -p ${{ secrets.GITHUB_TOKEN }}
+          docker login -u "${{ secrets.CI_DOCKER_USERNAME }}" -p "${{ secrets.CI_DOCKER_PASSWORD }}"
       - uses: actions/checkout@v2
       - run: make ci-sweep
       - name: prepare environment
         run: make environment-prepare
       - name: push image
         if: github.event_name != 'workflow_dispatch'
-        run: docker push docker.pkg.github.com/timberio/vector/environment:${{ github.sha }}
+        run: |
+          docker push $ENVIRONMENT_UPSTREAM
+          docker tag $ENVIRONMENT_UPSTREAM timberio/ci_image
```

Comment:
I wonder why are we adding another tag for the environment image. I was under the impression it was perfectly usable as-is. @Hoverbear wasn't it the intent behind the `environment` image in the first place?